README.md

```diff
@@ -51,11 +51,33 @@ where `options` is a plain `Object` with any of the following:
 and `simulator` is an object containing:
 
 * `binary`: path to the simulator binary
+* 'bin': an alias to `binary`
 * `profile`: path to the simulator profile
 * `pid`: process id
 * `process`: the actual process object
 * `port`: the port where the simulator is listening for debugging connections
 
+There is also a `startSimulator.all()` utility to launch many simulators at once:
+
+```javascript
+startSimulator.all(options)(simulatorOptions)
```

Comment:
That one *does* look perfectly promise-y though. It’s a less verbose API but feels less standard. Blarg, so torn!